src/app/presence-control/models/presence-control-entry.model.ts

```diff
@@ -43,7 +43,9 @@ export class PresenceControlEntry implements Searchable {
   }
 
   get presenceCategory(): PresenceCategory {
-    if (isUnapprovedAbsence(this.settings, this.confirmationState?.Key)) {
+    if (
+      isUnapprovedAbsence(this.settings, Number(this.confirmationState?.Key))
```

Comment:
Wenn kein `confirmationState` vorhanden ist, wäre das dann `NaN`... das ist wohl nicht ganz in deinem Sinn, oder? Wahrscheinlich wäre `isUnapprovedAbsence(this.settings, this.confirmationState && Number(this.confirmationState.Key))` eher, das was du willst.

Reply:
Merssi - ich hab das angepasst.